Uyanda.Coffee.Application/Features/BeverageManagement/Models/UserModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Uyanda.Coffee.Application.Features.BeverageManagement.Models
+{
+    public class UserModel
```

Comment:
Rather name this class _CustomerModel_. User has a connotation of being the user of the system, which customers are not.

So if the system ever has a login page, then the coffee company employees would be users.